### PR TITLE
Fix: Convert anchor tag to text node before generating PDF

### PR DESCRIPTION
**Pull request by jjjaspher**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix https://github.com/hlxsites/accenture-newsroom/issues/133

- https://github.com/hlxsites/accenture-newsroom/issues/133

Test URLs:
- Before: https://main--accenture-newsroom--hlxsites.hlx.page/news/2023/accenture-completes-acquisition-of-anser-advisory
- After: https://84836-PDF-Links--accenture-newsroom--hlxsites.hlx.page/news/2023/accenture-completes-acquisition-of-anser-advisory

### CODE REVIEW

File: blocks/aside/aside.js [hunk 1]
```diff
@@ -16,9 +16,22 @@ async function generatePDF(pageTitle) {
   const main = document.querySelector('main').cloneNode(true);
   const oHeroContainer = main.querySelector('.section.hero-container');
   const asideContainer = main.querySelector('.aside-container');
+  const oAllParagraphs = main.querySelectorAll('p');
```
[the remark this answers is not in the document]
Comment (jjjaspher) [reply blocks/aside/aside.js:19]: I just have a habit of appending the first letter of the data type of the variable. for example.
oObject = object or element
iInteger = integer
sString = string
aArray = array

but yeah will update those variables

[the remark this answers is not in the document]
Comment (jjjaspher) [reply blocks/aside/aside.js]: @amol-anand updated, please check once. thank you

File: blocks/aside/aside.js [hunk 2]
```diff
@@ -16,9 +16,22 @@ async function generatePDF(pageTitle) {
...
   oHeroContainer.remove();
   asideContainer.remove();
   const pageName = pageTitle.replace(/[^a-z0-9]/gi, '-');
+
+  oAllParagraphs.forEach((oParagraph) => {
+    const oAnchorElements = oParagraph.querySelectorAll('a');
```
[the remark this answers is not in the document]
Comment (jjjaspher) [reply blocks/aside/aside.js]: @amol-anand updated, please check once. thank you

File: blocks/aside/aside.js [hunk 3]
```diff
@@ -16,9 +16,22 @@ async function generatePDF(pageTitle) {
...
+    if (oAnchorElements.length === 0) {
+      return;
+    }
+    oAnchorElements.forEach((oElement) => {
+      const oInnerText = document.createTextNode(oElement.innerText);
```
[the remark this answers is not in the document]
Comment (jjjaspher) [reply blocks/aside/aside.js]: @amol-anand updated, please check once. thank you

File: blocks/aside/aside.js [hunk 4]
```diff
@@ -16,9 +16,22 @@ async function generatePDF(pageTitle) {
   const main = document.querySelector('main').cloneNode(true);
   const oHeroContainer = main.querySelector('.section.hero-container');
```
[the remark this answers is not in the document]
Comment (jjjaspher) [reply blocks/aside/aside.js]: @amol-anand updated, please check once. thank you

**Review by amol-anand — APPROVED** [review]

**Review by amol-anand — APPROVED** [review]

**Pull request merged**